client/test/unit/specs/utils/missionTypeFilter.spec.js

```diff
@@ -0,0 +1,47 @@
+import missionTypeFilter from '@/utils/missionTypeFilter';
+import jobFixture from '../fixtures/job.fixture';
+
+const buildJobWithMissionType = (nature) => {
+  const jobWithMissionType = jobFixture();
+  jobWithMissionType.project.nature = nature;
+  return jobWithMissionType;
+};
+
+describe('Unit | Utils | MissionType Filter', () => {
+  const deliveryJob = buildJobWithMissionType('delivery');
+  const consultingJob = buildJobWithMissionType('consulting');
+  const trainingJob = buildJobWithMissionType('training');
+  const allJobs = [deliveryJob, deliveryJob, consultingJob, trainingJob];
+
+  it('should return the 2 delivery jobs when missionType has Delivery', () => {
```

Comment:
Valable ici et pour les 3 prochains tests.
C'est "is Delivery" plutôt ?
Parce que has = contient, alors que ton test c'est quand il n'y a **que** Delivery

Reply:
c'est un array, donc j'ai mis "has"